maven-plugin/src/main/java/org/wildfly/galleon/maven/ArtifactListBuilder.java

```diff
@@ -16,6 +16,8 @@
  */
```

Comment:
I don't think that this class needs to be updated.

Reply:
fixed